packages/multi-test/src/contracts.rs

```diff
@@ -130,8 +136,8 @@ where
     }
 }
 
-impl<T1, T2, T3, E1, E2, E3, C, T4, E4, E5, T6, E6>
-    ContractWrapper<T1, T2, T3, E1, E2, E3, C, T4, E4, E5, T6, E6>
+impl<T1, T2, T3, E1, E2, E3, C, Q, T4, E4, E5, T6, E6>
+    ContractWrapper<T1, T2, T3, E1, E2, E3, C, Q, T4, E4, E5, T6, E6>
```

Comment:
Type complexity anyone?

Reply:
Are you suggesting simplifying this somehow? I'd definitely like to, I just have no idea how as of right now.

Reply:
No. Just couldn't help noticing the sheer number of type params.

Reply:
Ah, yeah, it's... something :grin: